engine_controller_node:
  ros__parameters:
    steps_to_full_circle: 240 # TODO(PriestOfAdanos): move to common_config.yml
    direction_pin: 20
    step_pin: 21
    prod_mode: True
    delay: .0208
    parent_frame_id: 'base_link'
    child_frame_id: 'top'
    translationXYZ: 
      - 0.0
      - 0.0
      - 0.0